client/yarn-error.log

```diff
@@ -0,0 +1,10646 @@
+Arguments: 
```

Comment:
yarn-error.log file should not be pushed to master. add it in .gitignore if it is not there already.